### ISSUE TITLE
Fix Travis build

### ISSUE THREAD

**Issue by mikegerber** [mikegerber/my_ocrd_workflow#18]
https://travis-ci.org/github/mikegerber/my_ocrd_workflow/jobs/699616134

**Comment by mikegerber** [comment]
Replacing the `pip3 check` with a `pipdeptree -w fail` fixed the problem with TensorFlow 2 somehow ... The remaining (and new) problem is that the test data does not validate using the newest `ocrd workspace validate`.

**Comment by mikegerber** [comment]
~~~
<report valid="false">

  <warning>pc:PcGts/@pcGtsId differs from mets:file/@ID: "" !== "OCR-D-IMG-BINPAGE_00000024"</warning>

</report>

The command "$TRAVIS_BUILD_DIR/run" exited with 128.
~~~
ocrd_olena seems to generate bad IDs?

**Comment by mikegerber** [comment]
(Update to latest ocrd_olena did not help.)

**Comment by mikegerber** [comment]
I'll remove the validation steps until the fix for https://github.com/OCR-D/ocrd_olena/issues/60 is merged.

**Comment by mikegerber** [comment]
Now there are problems with 

1. the news ocrd_olena putting PAGE and images into the same filegroup (solution is immanent: https://github.com/cisocrgroup/ocrd_cis/pull/57#issuecomment-656336593) and
2. sbb_textline_detector somehow producing garbled IDs - but that may be related to 1

**Comment by mikegerber** [comment]
I ficed  pcGtsId issues here:
* ocrd_tesserocr  (https://github.com/OCR-D/ocrd_tesserocr/issues/135)
* sbb_textline_detector (https://github.com/qurator-spk/sbb_textline_detection/commit/8b01d9e671d1527bdf52983ddd0799bcb1f4feda)
* ocrd_calamari (https://github.com/OCR-D/ocrd_calamari/issues/40)

**Comment by mikegerber** [comment]
>     1. ocrd_olena >= 1.2.0 putting PAGE and images into the same filegroup (solution is immanent: [cisocrgroup/ocrd_cis#57 (comment)](https://github.com/cisocrgroup/ocrd_cis/pull/57#issuecomment-656336593)) and

This is fixed in [OCR-D core v2.13.0](https://github.com/OCR-D/core/releases/tag/v2.13.0).

**Comment by mikegerber** [comment]
After [fixing the Tesseract download](https://github.com/mikegerber/my_ocrd_workflow/commit/c2946395e315bcc2da1bdee9d5d007488e7b86e2) the build is green again 🎉
